### DISCUSSION TITLE
Can't loop through dayStyle props in Calendar component.

### DISCUSSION THREAD

**Discussion by raseltechnext** [mantinedev/mantine#4304] [Q&A]
### What package has an issue

@mantine/dates

### Describe the bug

I want to loop through inside dayStyle prop. It works perfect like,

```js
dayStyle={date =>
  DateFormatter(date) === '2023-05-25'
    ? {
        backgroundColor: 'red',
        color: 'white'
      }
    : null
}
```

But if I want to loop through inside of this, this won't work,

```js
dayStyle={date =>
  ['2023-05-23'].forEach(item =>
    DateFormatter(date) === item
      ? {
          background: 'red'
        }
      : null
  )
}
```

I believe it should work like this.

### What version of @mantine/hooks page do you have in package.json?

^5.10.0

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
`.forEach` returns undefined. You need to use `.some` instead: `[/*dates*/].some(value => value === item) ? { /* styles */ } : null`
